### PR TITLE
Implemented header

### PR DESCRIPTION
**Pull request by Max2002**

### CODE REVIEW

File: src/components/button/Button.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <button className={className} type={type} onClick={onClick}>
+      {children}
+    </button>
+  );
+}
+
+Button.defaultProps = {
+  className: '',
+  onClick: () => {},
```
Comment (itzVa1kyrie) [inline src/components/button/Button.js]: delete

File: src/sections/header/Header.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import './header.scss';
```
Comment (itzVa1kyrie) [inline src/sections/header/Header.js]: First import components and important thing, then scss

File: src/sections/header/Header.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import clsx from 'clsx';
+import { useState } from 'react';
+import Button from '../../components/button/Button';
+import SignIn from '../signIn/SignIn';
+import { ReactComponent as Logo } from '../../assets/images/logo.svg';
+
+export default function Header() {
+  const [isSignIn, setIsSignIn] = useState(false);
+
+  const handleSignIn = () => {
```
Comment (itzVa1kyrie) [inline src/sections/header/Header.js]: now this can be in one line

File: src/sections/header/Header.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      <div className="logo">
+        <Logo />
+      </div>
+      <div className="menu">
+        <ul className="menu_nav">
+          <li className="menu-nav_item">Home</li>
+        </ul>
+        <div className="signIn">
+          <Button type="button" className="signIn_btn" onClick={handleSignIn}>
+            <svg
```
Comment (itzVa1kyrie) [inline src/sections/header/Header.js]: Put in single component

File: src/App.js [hunk 5]
```diff
@@ -1,3 +1,15 @@
+import Header from './sections/header/Header';
+import './assets/styles/general.scss';
+// import { BrowserRouter, Routes, Route } from 'react-router-dom';
```
Comment (GCoder-13) [inline src/App.js]: Delete comments

File: src/components/button/Button.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+export default function Button({ type, className, children, onClick }) {
+  return (
+    <button className={className} type={type} onClick={onClick}>
```
Comment (GCoder-13) [inline src/components/button/Button.js]: Use button from Bootstrap

File: src/sections/header/Header.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
+import './header.scss';
+import clsx from 'clsx';
+import { useState } from 'react';
+import Button from '../../components/button/Button';
+import SignIn from '../signIn/SignIn';
+import { ReactComponent as Logo } from '../../assets/images/logo.svg';
```
Comment (GCoder-13) [inline src/sections/header/Header.js]: Add import svg from /src/assets/icons

File: src/sections/header/header.scss [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    .menu_nav {
+      display: flex;
+      align-items: center;
+
+      .menu-nav_item {
+        color: rgba(0,0,0,.65);
+      }
+    }
+
+    .signIn {
```
Comment (GCoder-13) [inline src/sections/header/header.scss]: don't use camelcase in css

File: src/components/image/Image.js [hunk 9]
```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+
+export default function Image({ src, className }) {
```
Comment (GCoder-13) [inline src/components/image/Image.js]: why you need an Image component?

File: src/assets/styles/general.scss [hunk 10]
```diff
@@ -0,0 +1,33 @@
+@import 'variables';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
```
Comment (GCoder-13) [inline src/assets/styles/general.scss]: add container to module style

File: src/sections/header/header.module.scss [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+  width: 100%;
+}
+.menuNav {
+  display: flex;
+  align-items: center;
+}
+.menuNavItem {
+  color: rgba(0,0,0,.65);
+}
+.signIn {}
```
Comment (GCoder-13) [inline src/sections/header/header.module.scss]: signIn is empty. Delete

File: src/sections/header/header.module.scss [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+.header {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  position: fixed;
+  right: 0;
+  left: 0;
+  top: 0;
+  padding: 20px 40px;
+  background: #fff;
```
Comment (GCoder-13) [inline src/sections/header/header.module.scss]: use variable

File: src/sections/header/header.module.scss [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+  box-shadow: 0 0 10px 0 rgba(51, 51, 51, .33%);
```
Comment (GCoder-13) [inline src/sections/header/header.module.scss]: use variable

File: src/sections/header/header.module.scss [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+  align-items: center;
+  margin-left: 40px;
+  width: 100%;
+}
+.menuNav {
+  display: flex;
+  align-items: center;
+}
+.menuNavItem {
+  color: rgba(0,0,0,.65);
```
Comment (GCoder-13) [inline src/sections/header/header.module.scss]: use variable

File: src/sections/header/header.module.scss [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+}
+.signIn {}
+.signInBtn {
+  display: flex;
+  align-items: center;
+  color: #1890ff;
```
Comment (GCoder-13) [inline src/sections/header/header.module.scss]: variable

File: src/sections/header/header.module.scss [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+  }
+}
+.blurBlock {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: #000;
```
Comment (GCoder-13) [inline src/sections/header/header.module.scss]: variable

File: src/assets/icons/index.js [hunk 17]
```diff
@@ -0,0 +1,5 @@
+import { ReactComponent as LogoSvg } from './logo.svg';
+import { ReactComponent as SignInSvg } from './sigIn.svg';
+
+export default LogoSvg;
```
Comment (GCoder-13) [inline src/assets/icons/index.js]: dont use default export

File: src/sections/signIn/index.js [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+  password: '',
+};
+
+export default function SignIn() {
+  return (
+    <div className={st.modalWindow}>
+      <div className={st.title}>Sign In</div>
+      <Formik
+        initialValues={initialValues}
+        validate={(values) => {
```
Comment (GCoder-13) [inline src/sections/signIn/index.js]: Move the function up

File: src/sections/signIn/index.js [hunk 19]
```diff
@@ -0,0 +1,62 @@
+import { Formik, Form, Field, ErrorMessage } from 'formik';
```
Comment (GCoder-13) [inline src/sections/signIn/index.js]: delete component

Comment (GCoder-13) [reply src/sections/signIn/index.js]: its next task

File: src/App.js [hunk 20]
```diff
@@ -1,3 +1,10 @@
+import Header from './sections/header';
+import './assets/styles/general.scss';
+
 export default function App() {
-  return <div className="App" />;
+  return (
+    <div className="container">
```
Comment (GCoder-13) [inline src/App.js]: module style

**Pull request merged**